### PR TITLE
Discussion piece

### PR DESCRIPTION
**Pull request by baseLogiK**
My uninspired comment.

### CODE REVIEW

**Comment by efleming969** [comment]
this looks good

**Comment by vkamineni** [comment]
Agreed

**Comment by joshcodes** [comment]
This seems a bit uninspired. Maybe the ethos needs rethought.

**Comment by charlesryan1** [comment]
Commenting on commenting.

**Comment by baseLogiK** [comment]
Ethos, pathos, or logos?

**Comment by maverick** [comment]
Need to capitalize your "t" @efleming969 :)

File: README.md [hunk 1]
```diff
@@ -2,4 +2,5 @@
 Joel - 
 Shruti - 
 VJ - 
+Justin -
```
Comment (efleming969) [inline README.md]: just to that line

File: README.md [hunk 2]
```diff
@@ -2,4 +2,5 @@
 Joel -
```
Comment (JOELELEAZER) [inline README.md]: Adding my change now

File: README.md [hunk 3]
```diff
@@ -2,4 +2,5 @@
 Joel - 
 Shruti - 
 VJ -
```
Comment (joshcodes) [inline README.md]: Please consider adding world emoji

**Pull request merged**